python/src/FittingTest_doc.i.in

```diff
@@ -305,6 +311,10 @@ BIC : float
 Notes
 -----
 This is used for model selection.
+
+If a :class:`~openturns.Distribution` is used, the likelihood is evaluated on the sample. 
+If a :class:`~openturns.DistributionFactory` is used, the `build` method is used to create the distribution, and the likelihood is then evaluated.
```

Comment:
```suggestion
If a :class:`~openturns.DistributionFactory` is used, the :meth:`~openturns.DistributionFactory.build` method is used to create the distribution, and the likelihood is then evaluated.
```
Not sure this works, but normally it should.

Reply:
Ok. Fixed.

Reply:
Arg, does not work:

![image](https://user-images.githubusercontent.com/31351465/111825091-58964c80-88e7-11eb-9858-43bce09d0afa.png)

Reply:
Curious. The :class: command did work, though. What is especially weird is that one of the apostrophe signs is written, but the other is not. Perhaps I used the wrong sign? Maybe if you try to play around with it, you will find the correct syntax.

Reply:
Oups. A damned backquote. Fixed now:
![image](https://user-images.githubusercontent.com/31351465/111826590-2a197100-88e9-11eb-983e-bd75569bde84.png)